focus_ios/views/usage.view.lkml

```diff
@@ -0,0 +1,202 @@
+view: usage {
+  derived_table: {
+    sql_trigger_value: SELECT CURRENT_DATE() ;;
+    sql:
+      SELECT
+        submission_date,
+        client_id,
+        days_created_profile_bits,
+        days_seen_bits,
+        days_since_created_profile,
+        days_since_seen,
+        first_seen_date,
+        country,
+        locale
+      FROM `telemetry.nondesktop_clients_last_seen`
```

Comment:
I see this table is also used for focus_android. It might make sense to have the `nondesktop_clients_last_seen` view in looker-hub instead of having it as a derived table here. Then the existing view could simply be extended by additional dimensions without having to manually maintain existing dimensions, like `country` or `submission_date`. I guess we could create a `nondesktop` namespace for this view and related ones.

The filtering could also be done in the explore by adding `sql_always_where`. This way we wouldn't need to use `derived_table` here, which is something we'd generally want to avoid (extending views is easier to maintain than custom written SQL, also changes, like adding fields, can propagate easier).

That being said, I'm also OK with merging this as is. Just things to keep in mind for future improvements.